### PR TITLE
MOLT Fetch 1.2.1 updates

### PR DESCRIPTION
**Pull request by taroface**
DOC-11632
DOC-11724
DOC-11971
DOC-11970
DOC-11986

- Update replication guidance and flags for MySQL and PG
- Add MySQL 5.7 guidance
- add --assume-role flag
- MOLT 1.2.1 release notes

### CODE REVIEW

File: src/current/releases/molt.md
Comment (ryanluu12345) [inline src/current/releases/molt.md:25]: Do you think it's worthwhile to show an example of using this flag in the docs above or do you think documenting it in "global flags" is enough here?

File: src/current/_includes/molt/replicator-flags.md
Comment (ryanluu12345) [inline src/current/_includes/molt/replicator-flags.md:31]: Isn't the `--stagingSchema` basically a STRING of a certain format? Why is it represented as "ATOM"? That's quite confusing.

File: src/current/molt/molt-fetch.md
Comment (ryanluu12345) [inline src/current/molt/molt-fetch.md:362]: Just a note that you technically do not need the `--pglogical-publication-name`. That's optional since we default to `molt_fetch`. But they should put the drop and recreate flag for sure in this mode.

This basically replaces the manual process.

**Review by ryanluu12345 — COMMENTED** [review]
LGTM for the pglogical and assume role portions (pending the discussion about if we should include an example). I'll let @noelcrl chime in on all MySQL portions.

File: src/current/molt/molt-fetch.md
Comment (noelcrl) [inline src/current/molt/molt-fetch.md:382]: What do you think about changing it to something similar to: "the initial run of either \`--mode replication-only\` or \`--mode data-load-and-replication\`", or "the initial run of replication." Technically, it could be either mode, but do we need exhaustive detail here?

**Review by ZhouXing19 — APPROVED** [review]
LGTM for parts that I touched.

File: src/current/releases/molt.md
Comment (taroface) [reply src/current/releases/molt.md:25]: There is actually an example in the flags table, so I think that should suffice!

File: src/current/_includes/molt/replicator-flags.md
Comment (taroface) [reply src/current/_includes/molt/replicator-flags.md:31]: The types are 1:1 from the original Replicator doc, as I didn't have time to verify whether we could use alternatives. But I'll definitely amend this one if you're confirming that it's a `STRING`.

Comment (taroface) [reply src/current/_includes/molt/replicator-flags.md:31]: I'll amend `--targetSchema` to a `STRING` type as well, while I'm here.

File: src/current/molt/molt-fetch.md
Comment (taroface) [reply src/current/molt/molt-fetch.md:351]: Nice catch, thanks!!

File: src/current/molt/molt-fetch.md
Comment (taroface) [reply src/current/molt/molt-fetch.md:382]: Amended and ended up turning this into its own section.

**Comment by taroface** [comment]
Thanks @noelcrl and @ryanluu12345, PTAL!

File: src/current/releases/molt.md
Comment (ryanluu12345) [reply src/current/releases/molt.md:25]: Ahh I see it. You're right. That should be clear enough for folks.

File: src/current/_includes/molt/replicator-flags.md
Comment (ryanluu12345) [reply src/current/_includes/molt/replicator-flags.md]: Thanks, this will be a lot clearer now. The example also shows the string format, which is helpful.

**Review by ryanluu12345 — APPROVED** [review]
Docs LGTM. I'll have @noelcrl chime in on his sections one last time. And once he gives the stamp, let's merge this :)

**Comment by taroface** [comment]
> Docs LGTM. I'll have @noelcrl chime in on his sections one last time. And once he gives the stamp, let's merge this :)

FYI - it also needs to pass Docs review as always.

File: src/current/molt/molt-fetch.md
Comment (kathancox) [inline src/current/molt/molt-fetch.md:45]: Did you want this line starting with "For MySQL 5.7..." to be indented the same level as "For MySQL 8.0"...? Right now, the 5.7 bullet is rendering inline with the flags.

File: src/current/releases/molt.md
Comment (kathancox) [inline src/current/releases/molt.md:25]:
```suggestion
- MOLT Fetch users now can use [`--assume-role`]({% link molt/molt-fetch.md %}#global-flags) to specify a service account for assume role authentication to cloud storage. `--assume-role` must be used with `--use-implicit-auth`, or the service account will be ignored. 
```

What do you think? I had to do a quick re-read to realize that "it" was referring to assume role.

**Review by kathancox — APPROVED** [review]
LGTM! Couple of minor, non-blocking comments.

File: src/current/releases/molt.md
Comment (taroface) [reply src/current/releases/molt.md:25]: Hmm I think we can say "the flag will be ignored" as that's simplest.

**Pull request merged**
